TileView: Add Getting Started (#7795)
DevExpress/devextreme-documentation: concepts/05 UI Components/TileView/05 Getting Started with TileView/20 Configure TileView Properties.md

    >
        <!-- ... -->
    </dx-tile-view>

##### Vue

    <!-- tab: App.vue -->
    <template>
        <DxTileView
            :data-source="tiles"
            :item-margin="16"
            :base-item-width="280"
            :base-item-height="160"
            :height="368"
            :active-state-enabled="false"
            item-template="tileTemplate"
        >
            <!-- ... -->
        </DxTileView>
    </template>

##### React

    <!-- tab: App.tsx -->
    export default function App(): JSX.Element {
        return (
            <TileView
                dataSource={tiles}
                itemMargin={16}
                baseItemWidth={280}
                baseItemHeight={160}
                height={368}
                activeStateEnabled={false}
                itemRender={TileRender}
            />
        );
    }

---

You can also configure the following properties to further customize the TileView component:

- **items[]**.[widthRatio](/Documentation/ApiReference/UI_Components/dxTileView/Configuration/items/#widthRatio)/**items[]**.[heightRatio](/Documentation/ApiReference/UI_Components/dxTileView/Configuration/items/#heightRatio)    

> Specify a multiplier for an item's width/height. Implement these properties to create tiles of different sizes.

Reviewer: ```suggestion
Adjust an item's width/height. Set these properties to create tiles of different sizes.
```
